be2-scala/src/test/scala/ch/epfl/pop/pubsub/graph/validators/ElectionValidatorSuite.scala

```diff
@@ -266,6 +270,48 @@ class ElectionValidatorSuite extends TestKit(ActorSystem("electionValidatorTestA
     system.actorOf(dbActorMock)
   }
 
+  private def mockDbForResultElection: AskableActorRef = {
+    val dbActorMock = Props(new Actor() {
+      override def receive: Receive = {
+        case DbActor.ReadLaoData(_) =>
+          sender() ! DbActor.DbActorReadLaoDataAck(laoDataForResultElection)
+        case DbActor.Catchup(_) =>
+          sender() ! DbActor.DbActorCatchupAck(List(MESSAGE_END_ELECTION_WORKING))
+        case DbActor.ReadSetupElectionMessage(_) =>
+          sender() ! DbActor.DbActorReadAck(Some(MESSAGE_SETUPELECTION_OPEN_BALLOT_WORKING))
+      }
+    })
+    system.actorOf(dbActorMock)
+  }
+
+  private def mockDbElectionNotEnded: AskableActorRef = {
+    val dbActorMock = Props(new Actor() {
+      override def receive: Receive = {
+        case DbActor.ReadLaoData(_) =>
+          sender() ! DbActor.DbActorReadLaoDataAck(laoDataForResultElection)
+        case DbActor.Catchup(_) =>
+          sender() ! DbActor.DbActorCatchupAck(List.empty)
+        case DbActor.ReadSetupElectionMessage(_) =>
+          sender() ! DbActor.DbActorReadAck(Some(MESSAGE_SETUPELECTION_OPEN_BALLOT_WORKING))
+      }
+    })
+    system.actorOf(dbActorMock)
+  }
+
+  private def mockDbWrongQuestionIds: AskableActorRef = {
+    val dbActorMock = Props(new Actor() {
+      override def receive: Receive = {
+        case DbActor.ReadLaoData(_) =>
+          sender() ! DbActor.DbActorReadLaoDataAck(laoDataForResultElection)
+        case DbActor.Catchup(_) =>
+          sender() ! DbActor.DbActorCatchupAck(List(MESSAGE_END_ELECTION_WRONG_ID))
```

Comment:
I reiterate with my previous comments about this mockDb. Also, why do you put `MESSAGE_END_ELECTION_WRONG_ID` and not simply `MESSAGE_END_ELECTION_WORKING`?